SocialFilter/sections/css_extended.txt

```diff
@@ -15,6 +15,7 @@ karapaia.com#?#.article-body > div[style^="text-align:center;"]:has(> iframe[src
 pokemongo-get.com#?#.post_bottom > div + div:has(> div.pg_board_post_button > i.fa.mr-5)
 psyche.co#?#div[class^="ideas__SocialShareBarWrapper-"] > div > p:has(> svg[data-cy$="-outline"])
 manga1001.club#?#.section.box:has(> .social_share_wrapper)
+appbank.net#?#.lets-push:contains(シェア)
```

Comment:
I guess simple `##.lets-push` to specific.txt is enough. Sorry if you know false positive by that and avoided.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/129381#discussion_r967863484
The reason is the same as this.
Am I worrying about it too much?

Reply:
If `appbank.net##.lets-push` is safe on all `appbank.net` pages, it'll be preferred.

Reply:
I will change rule to  `appbank.net##.lets-push`